src/pkg/services/m365/api/users.go

```diff
@@ -175,17 +175,16 @@ func (c Users) GetInfo(ctx context.Context, userID string) (*UserInfo, error) {
 	var (
 		// Assume all services are enabled
 		// then filter down to only services the user has enabled
-		userInfo = newUserInfo()
-
+		userInfo        = newUserInfo()
 		mailFolderFound = true
 	)
 
 	// check whether the user is able to access their onedrive drive.
 	// if they cannot, we can assume they are ineligible for onedrive backups.
 	if _, err := c.GetDefaultDrive(ctx, userID); err != nil {
-		if !clues.HasLabel(err, graph.LabelsMysiteNotFound) || clues.HasLabel(err, graph.LabelsNoSharePointLicense) {
-			logger.CtxErr(ctx, err).Error("getting user's drive")
-			return nil, graph.Wrap(ctx, err, "getting user's drive")
+		if !clues.HasLabel(err, graph.LabelsMysiteNotFound) && !clues.HasLabel(err, graph.LabelsNoSharePointLicense) {
+			logger.CtxErr(ctx, err).Error("getting user's default drive")
+			return nil, graph.Wrap(ctx, err, "getting user's default drive info")
```

Comment:
I'm still a little confused by this. Honestly determining if drives are enabled via SP licenses in a call to GetInfo which seems mostly about individual users seems odd

Reply:
If the tenant has no SP license, then that's the error we get when retrieving user's drives, instead of a MySiteNotFound error.  We're just catching what graph gives us.  In the meantime, it's easier to translate that response to "the user cannot access drives" instead of "this user has a tenant-level error".